proposal/calc-functions.md

```diff
@@ -222,15 +303,20 @@ This algorithm takes a calculation `calc` and returns a number or a calculation.
 
       [definitely-incompatible]: ../spec/types/number.md#possibly-compatible-numbers
 
-    * If `dividend` and `modulus` are mutually [compatible], let `result` be the
-      result of `dividend % modulus`.
+    * If `dividend` and `modulus` are mutually [compatible]:
 
-      [compatible]: ../spec/types/number.md#compatible-units
+      * Let `result` be the result of `dividend % modulus`.
 
-    * If `calc`'s name is `"rem"`, and if `dividend` is positive and `modulus`
-      is negative or vice versa, set `result` to `mod - dividend`.
+        [compatible]: ../spec/types/number.md#compatible-units
 
-    * Return `result`.
+      * If `calc`'s name is `"rem"`, and if `dividend` is positive and `modulus`
+        is negative or vice versa:
+
+        * If `modulus` is infinite, return `dividend`.
+        * If `result` [exactly equals] 0, return `-result`.
+        * Otherwise, return `result - dividend`.
```

Comment:
I see I found this confusing before (the subtraction) but that's because of my lack of understanding of how % works when the signs are opposite (and because dart `%` behaves differently I think)

IIUC:
```
70 % 20 = 10
-70 % 20 = 10
70 % -20 = -10
-70 % -20 = -10
```
Is that correct?

Reply:
That's right. In Sass and CSS, the sign of the result of modulo is always the same as the sign of the right-hand side.